packages/hevea/hevea.2.36/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: "Luc Maranget"
+homepage: "http://hevea.inria.fr/"
+bug-reports: "http://github.com/herd/hevea/issues/"
+doc: "http://hevea.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/maranget/hevea.git"
+license: ["QPL-1.0" "LGPL-2 with exceptions"]
```

Comment:
I'm not seeing the exception in the license for the LGPL, only for the QPL part.
```suggestion
license: ["QPL-1.0 WITH OCaml-LGPL-linking-exception" "LGPL-2.0-or-later"]
```

Reply:
Note: after reviewing the LICENSE terms, I believe that it is actually LGPL-2.0-only, not -or-later, so I changed the package accordingly. The LGPL license only applies to "binary distributions of the package" anyway, so it is unclear to me that they actually apply to distribution through opam, but oh well, better be safe than sorry.

Reply:
>  I believe that it is actually LGPL-2.0-only, not -or-later, so I changed the package accordingly.

Are you sure? The license file does say:
```
    This library is free software; you can redistribute it and/or
    modify it under the terms of the GNU Library General Public
    License as published by the Free Software Foundation; either
    version 2 of the License, or (at your option) any later version.
```

Reply:
Ah indeed. In that case let's remove mentions of the LGPL entirely given it doesn't apply to any of the opam versions.